part2/stages/Functions/install-main

```diff
@@ -591,7 +594,18 @@ create_efi_boot_entries()
 
 copy_to_esp()
 {
-    local ESP=$(sfdisk -l -q | grep "EFI System" | awk '{ print $1 }')
+    # TARGET_DISK not defined in upgrades; locate our
+    # installation disk manually.
+    if [ -z "${TARGET_DISK}" ]; then
+        local part="$(vgs xenclient --noheadings -o pv_name | sed 's/\s*//g' | sed 's|^/dev/||g')"
```

Comment:
A since sed instance can run multiple commands like this: `sed -e 's/\s*//g' -e 's|^/dev/||'`